x/gamm/twap/keeper.go

```diff
@@ -0,0 +1,8 @@
+package twap
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type twapkeeper struct {
+	// storeKey     sdk.StoreKey
```

Comment:
deadcode

Reply:
Lets keep this for now, basically have to add back that line once more logic exists